okhttp/src/test/java/okhttp3/KotlinSourceCompatibilityTest.kt

```diff
@@ -473,7 +473,7 @@ class KotlinSourceCompatibilityTest {
 
   @Test @Ignore
   fun httpLoggingInterceptor() {
-    var interceptor: HttpLoggingInterceptor = HttpLoggingInterceptor()
+    var interceptor = HttpLoggingInterceptor()
```

Comment:
For this file only I want to keep explicit types so that if we break that we’ll catch it. (For example, we could inadvertently make `HttpLoggingInterceptor()` return an `Interceptor`.

Reply:
how?  Is that via the Companion as a factory?  Isn't HttpLoggingInterceptor() a constructor call?

I can revert, but Intellij flags these as a warning.

Reply:
Yeah, it’s a constructor today but doesn’t need to be.

Can you suppress the warning? This file is not supposed to be warning-free!